azure_monitor/src/azure_monitor/trace.py

```diff
@@ -17,13 +17,13 @@
 
 
 class AzureMonitorSpanExporter(SpanExporter):
-    def __init__(self, **options):
-        self.options = utils.Options(**options)
+    def __init__(self, options: "utils.Options"):
```

Comment:
I think this is a breaking change?

Reply:
It is. Before the parameter was a dict that was used to create the `Options`. Now, the parameter is directly the `Options`. If this is not desirable, I'm will gladly restore the old behaviour in a more clear way!

Reply:
Yes! I think we want to keep the instrumentation method the same way as to not have to force users to change their code.

Reply:
@lzchen brought back the old behaviour!